boards/xtensa/olimex_esp32_evb/olimex_esp32_evb-pinctrl.dtsi

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022 Henrik Brix Andersen <henrik@brixandersen.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32-pinctrl.h>
+#include <dt-bindings/pinctrl/esp32-gpio-sigmap.h>
+
+&pinctrl {
+	uart0_tx_gpio1: uart0_tx_gpio1 {
+		pinmux = <UART0_TX_GPIO1>;
+	};
+
+	uart0_rx_gpio3: uart0_rx_gpio3 {
+		pinmux = <UART0_RX_GPIO3>;
+	};
+
+	uart1_tx_gpio4: uart1_tx_gpio4 {
+		pinmux = <UART1_TX_GPIO4>;
+	};
+
+	uart1_rx_gpio36: uart1_rx_gpio36 {
+		pinmux = <UART1_RX_GPIO36>;
+	};
+
+	i2c0_sda_gpio16: i2c0_sda_gpio16 {
+		pinmux = <I2C0_SDA_GPIO16>;
+		drive-open-drain;
+		output-high;
+	};
+
+	i2c0_scl_gpio13: i2c0_scl_gpio13 {
+		pinmux = <I2C0_SCL_GPIO13>;
+		drive-open-drain;
+		output-high;
+	};
```

Comment:
SDA and SCL are swapped, if I interpret the schematic correctly.
```suggestion
	i2c0_sda_gpio13: i2c0_sda_gpio13 {
		pinmux = <I2C0_SDA_GPIO13>;
		drive-open-drain;
		output-high;
	};

	i2c0_scl_gpio16: i2c0_scl_gpio16 {
		pinmux = <I2C0_SCL_GPIO16>;
		drive-open-drain;
		output-high;
	};
```

Reply:
@martinjaeger Thank you for the thorough review! They were indeed swapped (which is kinda of funny since I had them correct in the first iteration, then - after having probed the signals using a logic analyser - somehow came to the conclusion that they needed to be swapped. Sigh).